### DISCUSSION TITLE
Fatal error: /usr/share/z-push/z-push-2.7.1/src/lib/utils/utils.php:1485

### DISCUSSION THREAD

**Discussion by buckets-dev** [Z-Hub/Z-Push#97] [Q&A] [answered]
I am currently running z-push v2.7.0, z-push zimbra-backend 72, php-fpm 8.3 on Ubuntu 20.04.6 LTS with Nginx installed.

I haven't been  able to upgrade to v2.7.1 - v2.7.3 due the following error:-

![Screenshot 2024-08-04 215546](https://github.com/user-attachments/assets/a20127e7-7996-4667-b72d-d5f062efe028)

"04/08/2024 21:17:17 [1703279] [ERROR] [user@email.au] LoopDetection->ProcessLoopDetectionPreviousConnectionFailed(): Command 'Sync' at 04/08/2024 21:17:14 with pid '1703279' terminated unexpectedly or is still running.
04/08/2024 21:17:17 [1703279] [ERROR] [user@email.au] Please check your logs for this PID and errors like PHP-Fatals or Apache segmentation faults and report your results to the Z-Push dev team.
04/08/2024 21:17:17 [1703280] [WARN] [user@email.au] Mobile loop detected! Messages sent to the mobile will be restricted to 1 items in order to identify the conflict
04/08/2024 21:17:17 [1703280] [FATAL] [user@email.au] Fatal error: /usr/share/z-push/z-push-2.7.1/src/lib/utils/utils.php:1485 - Uncaught Error: Call to undefined function datefmt_create() in /usr/share/z-push/z-push-2.7.1/src/lib/utils/utils.php:1485
Stack trace:
#0 /usr/share/z-push/z-push-2.7.1/src/lib/core/streamer.php(467): Utils::FormatDateUtc()
#1 /usr/share/z-push/z-push-2.7.1/src/lib/core/streamer.php(321): Streamer->formatDate()
#2 /usr/share/z-push/z-push-2.7.1/src/lib/core/streamimporter.php(171): Streamer->Encode()
#3 /usr/share/z-push/z-push-2.7.1/src/lib/default/diffbackend/exportchangesdiff.php(167): ImportChangesStream->ImportMessageChange()
#4 /usr/share/z-push/z-push-2.7.1/src/lib/request/sync.php(1199): ExportChangesDiff->Synchronize()
#5 /usr/share/z-push/z-push-2.7.1/src/lib/request/sync.php(956): Sync->syncFolder()
#6 /usr/share/z-push/z-push-2.7.1/src/lib/request/requestprocessor.php(116): Sync->Handle()
#7 /usr/share/z-push/z-push-2.7.1/src/index.php(107): RequestProcessor::HandleRequest()
#8 {main}
  thrown (1)"

Help please?

**Comment by matidau** [comment, marked as answer]
Try

```
sudo apt install php8.3-intl
```

**Reply by buckets-dev** [reply]
Legend! This fixed the issue.

**Comment by buckets-dev** [comment]
Resolved.
